src/pocket_save.py

```diff
@@ -10,7 +10,7 @@ def main(wf):
     current_app = os.popen(
         """ osascript -e 'application (path to frontmost application as text)' """).readline().rstrip()
     if current_app in ['Google Chrome', 'Safari']:
-        if not add_item(get_browser_item(current_app)):
+        if not add_item(get_browser_item(current_app, wf.args)):
```

Comment:
Oops :)
Sorry, that must have been a copy/paste error. I programmed it directly in my workflow directory and later copied the changes to my local repo. I also didn't consider the calling add_item with a link from the clipboard. It's good that you added that too.